monorepo_commit

```diff
@@ -1 +1 @@
-d216e7599f499adec2e0200e599f601ea5424cde
+f2ed7dd9fa5cf11f9ffa773921cf4d1f99a5948d
```

Comment:
@gastonponti This commit results in the correct gas values: https://github.com/celo-org/celo-monorepo/commit/f2ed7dd9fa5cf11f9ffa773921cf4d1f99a5948d

However, it's still a bit unclear where those values come from. Our backwards calculation resulted in the commit linked above.

I also tried understanding it better by tracing those transaction. However this ended in even more questions 😆 
- When calling the normal tracer, it reports a gas usage of 102953.
- When using the call tracer a gas usage of 31321 is reported on the outermost frame
  - When adding the intrinsic tx costs to that value I get 102,321, which gives a difference of 632 that I cannot explain (yet)

Reply:
> This gives me 35300, not 34300. Is it a typo?

Yes, probably mixed something up.

Reply:
Looks like you haven't added memory data costs to the intrinsic costs
`31,321 + 21,000 + 50,000 = 102,321` but then there is also
```
gas_cost += 4 * bytes_zero: gas added to base cost for every zero byte of memory data
gas_cost += 16 * bytes_nonzero: gas added to base cost for every nonzero byte of memory data
```
that need to be added.